.env.example

```diff
@@ -3,6 +3,9 @@ DEV_MONGO_URI=mongodb://127.0.0.1:27017/booking
 DEV_MONGO_USER=booking-user
 DEV_MONGO_PASS=booking-pass
 
+# NOTIFICATION Package
+DEV_EMAIL_ACCOUNT=
+DEV_EMAIL_PASS=
```

Comment:
These changes are done over a shared file?

Reply:
The env variables should have the same name independently the environment (develop, prod or any other). If we follow this pattern we will have the same variables for each environment that we have isn't?